src/model/clean.ts

```diff
@@ -11,18 +12,14 @@ export const bigBullets = (s: string) =>
 const { window } = new JSDOM('');
 const DOMPurify = createDOMPurify(window);
 
-export const clean = compose(
-	bigBullets,
-	(s: string) =>
-		minify(s, {
-			collapseWhitespace: true,
-			minifyCSS: true,
-			minifyJS: true,
-		}),
-	// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-	DOMPurify.sanitize,
+export const clean = compose(bigBullets, (s: string) =>
+	minify(s, {
+		collapseWhitespace: true,
+		minifyCSS: true,
+		minifyJS: true,
+	}),
 );
 
-export const sanitiseHTML = (html: string, opts?: any): string =>
+export const sanitiseHTML = (html: string | ReactNode, opts?: any): string =>
```

Comment:
I wasn't sure if this would work? `DOMPurify.sanitize` only works on strings I think.

Reply:
TS was complaining about it potentially not being a string, but I can take another look.

Reply:
okay yeah that would be great. I just wasn't sure if DOMPurify would blow up if passed something other than a string. The docs only mention string at least.

Reply:
Actually yes - I had misunderstood this - the problem was that it was potentially undefined. I have refactored to make sure TS knows it isn't.